dail/postponed-expedited-snap-verifications.vbs

```diff
@@ -71,8 +71,6 @@ IF datediff("D", SNAP_application_date, fifteen_of_appl_month) >= 0 Then							'
 	progs_closed = progs_closed & "SNAP/"
 	SNAP_last_REIN_date = dateadd("d", 30, SNAP_application_date)
 	SNAP_followup_text = ", after which a new Combined Application Form (CAF) is required (expedited SNAP closing for postponed verification not returned)."
-	'To do - where is cash_check coming from?
-	IF cash_check <> 1 THEN intake_date = dateadd("d", 1, SNAP_last_REIN_date)			        'if cash is not being closed the intake date needs to be the day after the rein date
```

Comment:
Since we removed the lines related to the SNAP REIN date, I also removed this line as it is used to generate intake date. Let me know if this was incorrect.